runtime/vm/ValueTypeHelpers.cpp

```diff
@@ -63,4 +63,10 @@ valueTypeCapableAcmp(J9VMThread *currentThread, j9object_t lhs, j9object_t rhs)
 	MM_ObjectAccessBarrierAPI objectAccessBarrier(currentThread);
 	return VM_ValueTypeHelpers::acmp(currentThread, objectAccessBarrier, lhs, rhs);
 }
+
+BOOLEAN
+isClassRefQtypeHelper(J9Class *cpContextClass, U_16 cpIndex)
```

Comment:
I'd rather this wasn't named "helper" - I don't think there would be a conflict with the inline C++ version.